lib/gds_api/publishing_api_v2.rb

```diff
@@ -324,6 +324,19 @@ def get_editions(params = {})
     get_json(get_editions_url(params))
   end
 
+  def get_paged_editions(params = {})
+    Enumerator.new do |yielder|
+      next_link = get_editions_url(params)
+      while next_link
+        yielder.yield begin
+          response = get_json(next_link)
```

Comment:
I wonder if it's more intuitive here to iterate over the results and then yield them one by one? Or perhaps include another convenience method that does that? What do you think?